session/promise/storage_test.go

```diff
@@ -47,7 +47,7 @@ var (
 func Test_Storage_IssuesOneForUnknownID(t *testing.T) {
 	ms := newMockStorage(nil)
 	s := NewStorage(ms)
-	res, err := s.GetNewSeqIDForIssuer(id)
+	res, err := s.GetNewSeqIDForIssuer(id, id, id)
```

Comment:
`(id, id, id)` is scary stuff. :)

Reply:
I'll refactor the tests a bit to look a bit less intimidating :D